learners/setup.md

```diff
@@ -2,53 +2,154 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+The vantage6 has requirements on hardware and software to run its server and node instances. As for the detailed requirements, please refer to the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/requirements.html).
 
-## Data Sets
+For now, we just need to focus on the software setup.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+## Software setup
 
-## Software Setup
+This lesson requires the following software to be installed on your system:
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Operating system: Ubuntu 18.04+ , MacOS Big Sur+, or Windows 10+
+- Docker (always latest version)
+- Conda (latest version)
+- Python
+    - Python v3.10 for vantage6 version 3.8.0 or higher
+    - Python v3.7 for other lower versions of vantage6
+- Python packages:
+    - jupyterlab
+    - vantage6
+    - vantage6-client
+    - vantage6-algorithm-tools
 
-### Details
+### Docker
 
-Setup for different systems can be presented in dropdown menus via a `spoiler`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+We recommend installing [Docker Desktop](https://www.docker.com/products/docker-desktop/) on your system, which is the easiest way to install Docker.
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
+You can follow the official instructions to install Docker Desktop on [MacOS](https://docs.docker.com/desktop/install/mac-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/), or [Linux](https://docs.docker.com/desktop/install/linux-install/).
 
```

Comment:
Could you add a note here that in case the Linux user does not use *Docker Desktop* but the *Docker Engine* it should be able to run *Docker containers* without `sudo`, so checkout the post installation steps for linux: https://docs.docker.com/engine/install/linux-postinstall/

( I think linux user probably already have the Docker Engine installed, so they probably want to do it without Desktop)